src/taskpane/components/MergeComponents/MergeConflict.js

```diff
@@ -11,7 +11,7 @@ async function selectCell(e, sheet, cell) {
     runSelectCell(sheet, cell)
 }
 
-export default class MergeConflict extends React.Component {
+export default class MergeConflict extends React.PureComponent {
```

Comment:
Sorry, I should have been more clear. A functional component contrasts w/ a class component, so this line actually should read:

```
export default function MergeConflict(props) {
```

and then you can:
1. remove the constructor
2. remove all the `this.`

And generally remove like 10 lines of code or so.